src/plugins/tools/itemThemeSwitcher/itemThemeSwitcher.js

```diff
@@ -259,6 +260,7 @@ export default pluginFactory({
      * Called during the runner's destroy phase
      */
     destroy: function destroy() {
+        this.init();
```

Comment:
Could you please explain why it's needed? Can there be a simpler solution to resetting the values?

Reply:
Well, in init function you have the reset, and for avoid refactor a lot of stuff, this is the simple way to go. You cannot access to any variables because are inside of init function instead at plugin level

Reply:
And refactor more things, apart of go outside of scope of task, can add some bugs in future when regression tests that we dont found now

Reply:
I see your point.

If you think some refactoring is needed, could you please open a technical task to cover it?

Reply:
https://oat-sa.atlassian.net/browse/TBD-36

Reply:
But every file in currentTao needs refactor, I don't see the benefit for just refactor 1 file. Not benefit for us and not for customer

Reply:
It should be continuous work. That's why I initially suggested to apply a refactoring right away, boy scout rule.
Otherwise it has low chances of being prioritized.